src/main/java/uk/gov/ons/census/casesvc/service/UacProcessor.java

```diff
@@ -111,13 +150,23 @@ public void emitUacUpdatedEvent(UacQidLink uacQidLink, Case caze, boolean active
       uac.setRegion(caze.getRegion());
     }
 
-    Payload payload = new Payload();
-    payload.setUac(uac);
+    PayloadDTO payloadDTO = new PayloadDTO();
+    payloadDTO.setUac(uac);
     ResponseManagementEvent responseManagementEvent = new ResponseManagementEvent();
-    responseManagementEvent.setEvent(event);
-    responseManagementEvent.setPayload(payload);
+    responseManagementEvent.setEventDTO(eventDTO);
+    responseManagementEvent.setPayloadDTO(payloadDTO);
 
     rabbitTemplate.convertAndSend(
         outboundExchange, UAC_UPDATE_ROUTING_KEY, responseManagementEvent);
+
+    return payloadDTO;
+  }
+
+  private String convertPayloadDTOToJson(PayloadDTO payloadDTO) {
+    try {
+      return objectMapper.writeValueAsString(payloadDTO);
+    } catch (JsonProcessingException e) {
+      throw new RuntimeException("Failed converting PayloadDTO To Json");
```

Comment:
might want to include the `JsonProcessingException` when you construct the RuntimeException so that we get a better stacktrace

Reply:
done in method's new location